frontend/src/components/EditRecipe/EditRecipeForm.js

```diff
@@ -100,14 +114,16 @@ const EditRecipeForm = ({ history }) => {
         <div className="form-title col-4 offset-4">
           <label className="detail-labels" htmlFor="title">
             Recipe Title
-            <input className="form-control input-sm recipe-details" id="title" type="text" name="title" value={title} onChange={e => setTitle(e.target.value)} />
+            <input className="form-control input-sm recipe-details" id="title" type="text" name="title" value={title} onChange={e => setTitle(e.target.value)} required />
           </label>
+          <div className="errorMsg">{title ? '' : titleError}</div>
```

Comment:
I think you can just do `{titleError}`, because the `titleError` is an empty string by default.